packaging/deb/heimdallcli/DEBIAN/control

```diff
@@ -2,11 +2,11 @@ Source: heimdallcli
 Version:
 Section: develop
 Priority: standard
-Maintainer: Polygon <release-team@polygon.technology>
+Maintainer: Shibarium <team@shibariumtech.com>
```

Comment:
is this email correct?

Reply:
we dont have this email.. but we need to remove polygon name so we added ours